spec/test_app_templates/lib/generators/test_app_generator.rb

```diff
@@ -70,6 +70,12 @@ def relax_routing_constraint
     gsub_file 'config/initializers/arkivo_constraint.rb', 'false', 'true'
   end
 
+  # def run_action_cable_within_test_app
```

Comment:
minor thing: unless you plan on bringing this method back, I recommend deleting it and letting the revision control keep your backup!

Reply:
Yep, that's already in this branch. You may be looking at a change in an earlier commit.